gradle.properties

```diff
@@ -18,4 +18,13 @@ android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
 # Kotlin code style for this project: "official" or "obsolete":
-kotlin.code.style=official
\ No newline at end of file
+kotlin.code.style=official
+
+
+# App Config
+FLAVORS_DIMENSION=deuvox
+
+
+# App URL
+BASE_URL="https://private-anon-f649764b47-deuvox.apiary-mock.com/"
+BASE_URL_DEV="https://private-anon-f649764b47-deuvox.apiary-mock.com/"
```

Comment:
never put url on this git, i'm gonna add some ignore properties

Reply:
noted 👍